README.md

```diff
@@ -1 +1,186 @@
-# Leaderboard
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <h3><b>Leaderboard</b></h3>
+
+</div>
+
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+# 📖 [Leaderboard] <a name="about-project"></a>
+
+
+**[Leaderboard]** Is a personal priject that alows users to add and remove scores!
+
+## 🛠 Built With <a name="built-with"></a>
+
+HTML5, CSS3, JavaScript, Git and Github
+
+### Tech Stack <a name="tech-stack"></a>
+
+
+<details>
+  <summary>Client</summary>
+  <ul>
+    <li><a href="https://lenguajehtml.com/html/">HTML5</a></li>
+    <li><a href="https://lenguajecss.com/">CSS3</a></li>
+  </ul>
+</details>
+
+<details>
+  <summary>Server</summary>
+  <ul>
+    <li><a href="https://code.visualstudio.com/">Visual Studio Code</a></li>
+  </ul>
+</details>
+
+
+### Key Features <a name="key-features"></a>
+
+
+- **[Best organization]**
+- **[Professional documentation]**
+- **[Clean Working]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
```

Comment:
- [ ] For a more professional look, instead of living the live demo section blank, please leave a message like `Demo coming soon` or remove the section completely.